1120CDFSNCN29ARED/grupo-11
# GS1: Add home page

In this PR the home page was designed. 

How to test:
- Test the home page in responsive mode. Check the media queries to see the changes.
